common/projectconfig/projectconfig.go

```diff
@@ -47,6 +47,26 @@ func (c Config) Root() string {
 	return filepath.Dir(c.Path)
 }
 
+// Validate checks that the configuration is valid.
+func (c Config) Validate() error {
```

Comment:
Do we need a test for this one, or is it covered by other tests that use `Create`?